GTFO-API/Patches/GearManager_Patches.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using Gear;
+using HarmonyLib;
+using Player;
+using UnhollowerBaseLib.Attributes;
+
+namespace GTFO.API.Patches
+{
+    [HarmonyPatch(typeof(GearManager))]
+    internal class GearManager_Patches
+    {
+        [HideFromIl2Cpp]
```

Comment:
HideFromIl2Cpp attribute isn't necessary, the patch doesn't get injected into Il2Cpp domain